sorted_db/db.go

```diff
@@ -136,6 +136,13 @@ func indexByte(s []byte, i, m int, c byte) int {
 	return -1
 }
 
+// Copies all bytes in s to a new destination buffer
+func makeCopy(s []byte) []byte {
```

Comment:
I'd rename `s` to `source` and `d` to `dest` just for extra clarity, then the comment really isn't even necessary.